_sass/components/wells.scss

```diff
@@ -91,3 +91,34 @@
     }
   }
 }
+
+.well-in-development {
+  align-items: center;
+  display: inline-flex;
+  font-size: 15px;
+  margin-bottom: $md;
+  padding: $xs $sm;
+
+  @include media(sm) {
+    width: 100%;
+  }
+
+  @include media(xxs) {
+    align-items: flex-start;
+    flex-direction: column;
+  }
```

Comment:
@iPhone SE size. Should these items be center aligned at very small sizes? Should we use a negative margin hack here to expand the width?

<img width="384" alt="screen shot 2017-05-17 at 11 11 03 am" src="https://cloud.githubusercontent.com/assets/5341618/26169189/a04a6b80-3af1-11e7-87a4-7046f1e98e6a.png">